modules/ROOT/pages/to-create-a-mule-application-project.adoc

```diff
@@ -72,11 +67,10 @@ For example, if the project contains a Database connector, the database driver f
 
 === Procedure
 
-. On the *Projects* page of Design Center, click the *Create* button and then click *Import Application*.
+. On the *Projects* page of Design Center, click the *Create new* button and then click *Import from File*.
 . Specify a name to give to the imported project.
-. Browse your filesystem for the `.jar` file that contains the project.
-// . Browse your filesystem for the `.jar` file that contains the project. Both Design Center and Studio export Mule-application projects in `.jar` files.
-. Click *Import*.
+. Click *Choose file*, then browse your filesystem for the `.jar` file that contains the project.
+. Click *Import and Create project*.
```

Comment:
Add
"You must own an account with permissions to create mule apps. If no .jar option is shown inside import dialog, then you're missing permissions, check your account administrator"
Due to the ability of importing jar files not being showed if you're missing the rights to do so

Reply:
Added the following:

To import a project from a `.jar` file, your account must have permissions to create Mule applications. If you do not have this permission, the option to import a jar file is not available in the *Import from File* dialog.